src/store/reducers/genres.js

```diff
@@ -13,17 +15,23 @@ const defaultState = {
 
 export const genres = (state = defaultState, { type, payload }) => {
   switch (type) {
-    case GET_GENRES_BY_ID_PENDING:
-      return { ...state, isLoading: true };
     case GET_GENRES_PENDING:
-      return { ...state, isLoading: true };
+      return { ...state };
     case GET_GENRES_SUCCESS:
       return {
         ...state,
         byId: normalizeData(payload.genres),
-        isLoading: false,
       };
     case GET_GENRES_ERROR:
+      return { ...state };
+    case GET_GENRES_BY_IDS_PENDING:
+      return { ...state, isLoading: true };
+    case GET_GENRES_BY_IDS_SUCCESS:
+      return {
+        ...state,
+        byId: normalizeData(payload.genres),
```

Comment:
in this line you override all genres, but we should add or update genres. so i think better solution will be:
```js
byId: { ...state.byId, ...normalizeData(payload.genres) },
```
WDYT?